collector/catalog.go

```diff
@@ -0,0 +1,17 @@
+package collector
+
+import (
+	client "github.com/rancher/types/client/management/v3"
+)
+
+const rancherCatalogURL = "https://git.rancher.io/charts"
+
+func FilterRancherCatalogs(catalogNames []string, catalogs *client.CatalogCollection) []string {
```

Comment:
What's the point of passing catalogNames? The one place we use this just uses an empty slice.

Reply:
Yea not needed, used to be before rewrite.